ui/src/pages/FamilyPlanning.jsx

```diff
@@ -0,0 +1,415 @@
+import {
+    Container,
+    Stack,
+    Button,
+    Grid,
+    Snackbar,
+    Typography,
+    Divider,
+    useMediaQuery,
+    CircularProgress,
+    Modal,
+  } from "@mui/material";
+  import { useEffect, useState } from "react";
+  import { useNavigate } from "react-router-dom";
+  import Layout from "../components/Layout";
+  import { getCookie } from "../lib/cookie";
+  import Tab from "@mui/material/Tab";
+  import TabContext from "@mui/lab/TabContext";
+  import TabList from "@mui/lab/TabList";
+  import TabPanel from "@mui/lab/TabPanel";
+  import { Box } from "@mui/material";
+  import { DesktopDatePicker } from "@mui/x-date-pickers/DesktopDatePicker";
+  import { MobileDatePicker } from "@mui/x-date-pickers/MobileDatePicker";
+  import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+  import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+  import Radio from "@mui/material/Radio";
+  import RadioGroup from "@mui/material/RadioGroup";
```

Comment:
Please remove unused variables (such as from line 26 to 29) to pass linting tests. Ensure you remove any unused variable within this file and all other files you made changes to, if any.